tests/test_consistency.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2021 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+import pathlib
+import sys
+
+
+if sys.version_info > (3, 11):
```

Comment:
`>= (3, 11)` I think?

Reply:
Indeed. I wonder which of our dependencies brings in `tomli` on Python 3.11